### ISSUE TITLE
Nvidia all Invalid Results

### ISSUE THREAD

**Issue by Bathmat** [mox-project/mox-stak#1]
I'm only getting `Invalid Results` with my Nvidia GPUs and CUDA. Using a mix of GTX-1050, 1060, and 970. They are working when I use OpenCL.

**Comment by Stargazer77** [comment]
Yes. I have a problem with this too! Its dont work with NVIDIA GPU(1070,1080ti)! How to fix it? Is there a working version?
